README.md

```diff
@@ -13,3 +13,22 @@ The *tutorials* folder contains walkthroughs which demonstrate basic Certifai fu
 ## Certifai Documentation
 
 Refer to the Cortex Certifai documentation: https://cognitivescale.github.io/cortex-certifai/docs/about for detailed information about Cortex Certifai.
+
+## Notebooks Table of Contents
+
+| Notebook Name | Description | Task Type | Model Types | Evaluation Types | Key Properties |
+| --- | --- | --- | --- | --- | --- |
+| [AnalyzeFeatureUsage.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/analyze_feature_usage) - Analyzing Feature Usage | Illustrates the use of Certifai to create and run an explanations scan of a locally defined model from first principles. It then analyzes and displays feature distribution of the counterfactuals.  |  Binary classification | Decision Tree <br /> Logistic Regression | Explanations |  Feature Occurrence <br /> Frequency by Model |
+| [BringInYourOwnModel.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/tutorials/bringing_in_your_own_model/part_one) - Building a scan programmatically with your own model | Part one of a tutorial that takes data scientists through model preparation and scan. This notebook/tutorial is also packaged as part of the toolkit and can be accessed at `examples/notebooks/BringingInYourOwnModel.ipynb`. | Binary Classification | Logistic Regression | Fairness  <br /> Explainability <br />  Robustness |   |
+| (certifai_multiclass_evaluation)[model_train_part1.ipynb,  certifai_multiclass_evaluation_part2.ipynb and others](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/multiclass) | Illustrates how to start the multiclass classification task type scan. | Multiclass Classification | Logistic Regression | Performance Metric: Accuracy  <br /> Explainability-Explanations <br /> Fairness | One-hot encoding <br /> Target encoded features |  
```

Comment:
IMO we should drop this one entirely.  It really probably shouldn't be in master at all the notebook I mean) because its a specific example for a specific customer who needs 3 or 4 different things at once and as a result is not good from a pedagogical perspective.  However, it's not in scope of this PR to make that wide a change.  For the purposes of this PR I'd simply drop this one from the table though.  At the very least it should be highly questionable that a notebook called 'multiclass' does not have multi-class as a key property